djtriggers/tests/test_models.py

```diff
@@ -0,0 +1,98 @@
+from datetime import timedelta
+from logging import WARNING
+
+import mock
+import pytest
```

Comment:
Can you add requirement files for both regular install and test? 

Have a look here on how to accomplish this:
https://github.com/vikingco/cl_payments/tree/master/deployment
https://github.com/vikingco/cl_payments/blob/master/setup.py (look at the for loops over the files)